appveyor.yml

```diff
@@ -56,8 +56,12 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
 
-    # We don't need to build the example recipe.
-    - cmd: if exist recipes\example rmdir recipes\example /s /q
+    # Find the recipes from master in this PR and remove them.
+    - cmd: cd recipes
+    - cmd: git fetch https://github.com/conda-forge/staged-recipes.git master
+    - cmd: |
+             for /f "tokens=*" %%a in ('git ls-tree --name-only FETCH_HEAD -- .') do rmdir /s /q %%a && echo Removing recipe: %%a
+    - cmd: cd ..
```

Comment:
Yeah, I saw some AppVeyor scripts that used this with `ps`. Maybe there is something significant there that I'm missing. Wondering if it is an AppVeyor subtlety and not related to Windows generally. Have seen similar oddities with exit codes on other CI's like [Travis CI]( https://docs.travis-ci.com/user/customizing-the-build/#Note-on-%24%3F ).